storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="persistent-storage-csi-group-snapshots"]
+= CSI volume group snapshots
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-group-snapshots
+
+toc::[]
+
+This document describes how to use volume group snapshots with supported Container Storage Interface (CSI) drivers to help protect against data loss in {product-title}. Familiarity with xref:../../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[persistent volumes] is suggested.
+
+include::modules/persistent-storage-csi-group-snapshots-overview.adoc[leveloffset=+1]
+
+include::modules/persistent-storage-csi-group-snapshots-limitations.adoc[leveloffset=+1]
+
+include::modules/persistent-storage-csi-group-snapshot-create-admin.adoc[leveloffset=+1]
+
+include::modules/persistent-storage-csi-group-snapshots-create.adoc[leveloffset=+1]
+
+include::modules/persistent-storage-csi-group-snapshots-restore.adoc[leveloffset=+1]
+
+== Additional resources
+* xref:../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#csi-volume-snapshots[CSI volume snapshots]
```

Comment:
Not sure, but if you're trying to link [here](https://docs.openshift.com/container-platform/4.17/storage/container_storage_interface/persistent-storage-csi-snapshots.html#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots), then I think this link should be like this? I think this is the ID the build error is complaining about. 
```suggestion
* xref:../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[CSI volume snapshots]
```